source/transfer_list.rst

```diff
@@ -907,23 +907,61 @@ This manifest contains the SPMC attribute node consumed by the SPMD
 
 This entry type holds an array of `entry_point_info_t` structures.
 `entry_point_info_t` is a TF-A-specific data structure [TF_BL31]_ used to
-represent the execution state of an image, that is, the state of general purpose
-registers, PC and SPSR.
+represent the execution state of an image; that is, the state of general purpose
+registers, PC, and SPSR.
 
 .. code:: c
 
     typedef struct entry_point_info {
```

Comment:
please void typedef. We actually are not currently using C in the spec, so things are written out in table form.

However, I would be OK with us using C instead, if others agree. But perhaps cross that bridge later, after this PR is applied?